plugins/qcheck-stm/test/testing_warnings.ml

```diff
@@ -0,0 +1,18 @@
+type 'a t = 'a array
```

Comment:
Do we need this, as we don’t want to compile it anyway?

Reply:
I've removed the file, which is indeed unnecessary.